river/utils/skmultiflow_utils.py

```diff
@@ -264,3 +264,26 @@ def check_random_state(seed):
     if isinstance(seed, np.random.RandomState):
         return seed
     raise ValueError(f'{seed} cannot be used to seed a numpy.random.RandomState instance')
+
+
+def round_sig_fig(x, significant_digits=2):
```

Comment:
What is the purpose of this?

Reply:
I changed this function to return the rounded value directly. Let me explain its functioning:

This function helps us select only the significant digits in a long float (given the desired number of digits). 

For instance: 

- `round_sig_fig(0.0002345, significant_digits=2) -> 0.0002`
- `round(0.0002345, ndigits=2) -> 0.0`

On the other hand,

- `round_sig_fig(0.255, significant_digits=2) -> 0.26`
- `round(0.255, 2) -> 0.26`

It's a nice trick to keep significant decimal places and avoid creating non-meaningful rounding while drawing/printing the trees' structures. 

I always used a fixed number of decimal places to exhibit numbers in tables and similar stuff. That was until I performed a project in partnership with researchers of vitreous materials. They asked the guys from my lab and me to round the numbers to their significant figures. At first glance, the result seemed odd to me (as the resulting number of floating points is not the same for every number), but after second thought I realized the representation becomes more compact and concise.

Reply:
Nice! And thanks for the explanation.